trustfall_wasm/src/trustfall_wasm.d.ts

```diff
@@ -51,6 +51,7 @@ export interface Adapter<T> {
 
 export class Schema {
     free(): void;
+    subtypes(type_name: string): Set<string>
```

Comment:
Could you tweak this to add the type signature information and match the surrounding style here?

Reply:
done in 85d5a64

Reply:
```suggestion
    subtypes(type_name: string): Set<string>;
```